analysis/tests/combiners_test.py

```diff
@@ -1117,13 +1216,167 @@ def test_create_accumulator(self, probability_to_keep, metric, expected):
              )),
     )
     def test_compute_metrics(self, acc, expected):
-        combiner = combiners.SumAggregateErrorMetricsCombiner([0.5])
+        combiner = combiners.SumAggregateErrorMetricsCombiner(
+            metrics.AggregateMetricType.SUM, [0.5])
         metric = combiner.compute_metrics(acc)
-        self.assertEqual(expected, metric)
+        assert_aggregate_error_metrics_equals(self, expected, metric)
 
         # Test that no type is np.float64
         self.assertTrue(_check_none_are_np_float64(acc))
 
 
+def assert_aggregate_error_metrics_accumulator_equals(
+        test: parameterized.TestCase,
+        expected: combiners.AggregateErrorMetricsAccumulator,
+        actual: combiners.AggregateErrorMetricsAccumulator):
+    test.assertEqual(expected.num_partitions, actual.num_partitions)
+    test.assertAlmostEquals(expected.data_dropped_l0,
+                            actual.data_dropped_l0,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.data_dropped_linf,
+                            actual.data_dropped_linf,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.data_dropped_partition_selection,
+                            actual.data_dropped_partition_selection,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.error_l0_expected,
+                            actual.error_l0_expected,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.error_linf_expected,
+                            actual.error_linf_expected,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.error_linf_min_expected,
+                            actual.error_linf_min_expected,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.error_linf_max_expected,
+                            actual.error_linf_max_expected,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.error_l0_variance,
+                            actual.error_l0_variance,
+                            delta=1e-5)
+    test.assertAlmostEquals(expected.error_variance,
+                            actual.error_variance,
+                            delta=1e-5)
+    test.assertEqual(len(expected.error_quantiles), len(actual.error_quantiles))
+    [
+        test.assertAlmostEqual(expected.error_quantiles[i],
+                               actual.error_quantiles[i],
+                               delta=1e-5)
+        for i in range(len(expected.error_quantiles))
```

Comment:
Nit: Pythonic style is using zip instead of indices:

`for exp, act in zip(expected.error_quantiles, actual.error_quantiles):`